generators/client-2/templates/src/main/webapp/app/admin/audits/_audits.service.ts

```diff
@@ -1,20 +1,21 @@
 import { Injectable } from '@angular/core';
+import { Http, Response } from '@angular/http';
+import { Observable } from 'rxjs/Rx';
+import 'rxjs/add/operator/map';
 
-AuditsService.$inject = ['$resource'];
+export interface AuditsData {
+    //TODO put the model returned here
```

Comment:
I have no idea of the model (not the modal) returned by the backend... I'll check this in java code